assets/js/base/utils/address.ts

```diff
@@ -25,10 +22,9 @@ export const isSameAddress = < T extends ShippingAddress | BillingAddress >(
 	address1: T,
 	address2: T
 ): boolean => {
-	return Object.keys( defaultAddressFields ).every(
-		( field: string ) =>
-			address1[ field as keyof T ] === address2[ field as keyof T ]
-	);
+	return Object.keys( ADDRESS_FIELDS_KEYS ).every( ( field: string ) => {
+		return address1[ field as keyof T ] === address2[ field as keyof T ];
+	} );
```

Comment:
This would previously check against all fields, it's changed to only check address.